docs/installation/opni/aiops.md

```diff
@@ -88,32 +47,76 @@ Opni AIOps offers log anomaly detection on user's workload logs with the followi
 Note that it requires an GPU available in the Opni cluster.
 
 ## Prerequisites
-
-The Opni cluster must have **Opni Logging enabled** and needs at least **1 GPU** attached.
+* Opni logging enabled
+* Enabling AIOps and the GPU Services
+* At least 1 NVIDIA GPU attached
 
 ## Getting Started
-![SetupWorkloads](/img/aiops/setup_workload_loganomaly.png)
+![SetupWorkloads](/img/aiops/initial_workload_configuration.png)
 
-1. Navigate to the Opni Admin dashboard and select the **Workloads** tab
+1. Within the Opni Admin Dashboard, once AIOps has been enabled, go to the Workload Insights Tab.
 2. Select a cluster
 3. Select the workloads you want
 4. Click **Update Watchlist** to submit the configuration
 
-Give it some time for the AI models to get ready. You can then navigate to Opni's Opensearch dashboard and [consume AI Insights](#consuming-ai-insights-from-opni).
+The UI will give a status of the progress of the model. 
+![Model Training Status](/img/aiops/opni_train_workload_model.png)
+
+Once the model is ready, you can then navigate to Opni's Opensearch dashboard and [consume AI Insights](#consuming-ai-insights-from-opni).
 
 ### Remove workloads
 
-![RemoveWorkloads](/img/aiops/remove_workload_loganomaly.png)
+![RemoveWorkloads](/img/aiops/remove_workload_insights.png)
 
 To remove the workloads you don't want anymore, simply uncheck the boxes and **Update Watchlist**.
 
+### Reset workloads
+
+To reset your watchlist, simply hit the button **Clear Watchlist**.
+
 </TabItem>
 </Tabs>
-</TabItem>
-<TabItem value="metric-anomaly-detection" label="Metric Anomaly Detection (coming soon)">
 
-</TabItem>
-<TabItem value="root-cause-detection" label="Root Cause Detection (coming soon)">
+## Enabling AI Services
```

Comment:
maybe we should put this in the front, like the beginning of the overview? bc this is the prerequisites of some action items in the tabs.